schemas/test/en/test_currency.json

```diff
@@ -93,10 +91,93 @@
                                         "maximum": {
                                             "value": 1000000
                                         }
+                                    },
```

Comment:
The inputs allowing decimal places are all size bound, so is it worth having an additional input like this one for decimal places at min/max range? ( I can't see one in any of the other updated tests )

Reply:
Yeah I don't see why not, I shall update `test_numbers` to test decimal boundaries 👍

Reply:
Update: so we can implement the 6 decimals in test_numbers when setting boundaries up to 6 however, there is currently a bug where it would not display past the fist 3 decimals. So if you entered the boundires as 1`0.00315` and `1000.1234` you would get in the range collum. 10 and 1000.123. 
![image](https://user-images.githubusercontent.com/113098086/235653385-ec87f8bc-ac75-4822-82e5-cfb802eb42fb.png)
![image](https://user-images.githubusercontent.com/113098086/235653590-4453f841-60cd-473c-9339-7aa5f319056e.png)

Reply:
Ahhh I see - so the range still works correctly but doesn't display properly. I wonder if that'd cause confusion for someone later if they don't remember this, might be best to stick to 3 for the range boundaries? though I'd still be in favour of having a field somewhere on the test that allows 6dp

Reply:
Yeah after looking it is out of scope for this card, but there is no ticket yet. You can make a card if you would like on the trello board using the `bug template` on the far left and put it in `Tech Analysis` section. Otherwise I do not mind popping it on the trello board.

Reply:
done!